src/qonnx/custom_op/channels_last/__init__.py

```diff
@@ -1,29 +1,6 @@
+# Importing registers CustomOps in qonnx.custom_op.channels_last domain
 from qonnx.custom_op.channels_last.batch_normalization import BatchNormalization
 from qonnx.custom_op.channels_last.conv import Conv
 from qonnx.custom_op.channels_last.max_pool import MaxPool
 
-# channels-last ops are defined by the underlying ONNX standard op
-# thus, we can define them for any version of the original op
-# so we emulate a custom op dictionary that mimics the support for any
-# {ChannelsLastOp}_vX instead of hardcoding what versions are supported
-
-
-class ChannelsLastCustomOpDict(dict):
-    def __init__(self):
-        self._custom_ops = {"Conv": Conv, "MaxPool": MaxPool, "BatchNormalization": BatchNormalization}
-
-    def __getitem__(self, key):
-        base_key = key.split("_v")[0]  # Extract base key (e.g., Conv from Conv_v13)
-        if base_key in self._custom_ops:
-            return self._custom_ops[base_key]
-        raise KeyError(f"Channels-last CustomOp '{key}' not found.")
-
-    def __contains__(self, key):
-        base_key = key.split("_v")[0]
-        return base_key in self._custom_ops
-
-    def keys(self):
-        return self._custom_ops.keys()
-
-
-custom_op = ChannelsLastCustomOpDict()
+__all__ = ["Conv", "MaxPool", "BatchNormalization"]
```

Comment:
this does not respect the semantics (channels-last ops being implicitly defined for any version of the standard op) in the deleted code, instead all channels-last ops will now be registered as v1, meaning that they will instantiate ai.onnx v1 standard nodes under the hood. can you address this?

if this is tricky to fix in the new system, one band-aid we can apply is instead registering these channels-last ops as v11 which is the current preferred ONNX opset version. in this way they would do something slightly more reasonable.